app/build.gradle

```diff
@@ -17,18 +18,56 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    sourceSets { main { resources.srcDirs = ['src/main/resources', 'src/main/proto'] } }
+}
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:3.4.0'
+    }
+    plugins {
+        javalite {
+            artifact = 'com.google.protobuf:protoc-gen-javalite:3.0.0'
+        }
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java:1.7.0'
+        }
+    }
+    generateProtoTasks {
+        all()*.plugins {
```

Comment:
I remember that I added this entry, but you still came back with the two you previously wrote. Did you check that we need all of those three entries?

Reply:
You added this entry? And who do you think you are!?